#!/usr/bin/env python3
PACKAGE = "autopilot"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("kp_pid",    double_t,0, "kp:0.03,ki:0.02 for us vehicle and kp:0.12 ,ki:0.1 for india vehicles",0.12,0,0.13)
gen.add("ki_pid", double_t, 0, "Donnot give value more than 0.13 for both ki and kp",0.1, 0,0.13)

exit(gen.generate(PACKAGE, "autopilot", "pid_"))